use_of_fgetc.c

```diff
@@ -0,0 +1,21 @@
+#include<stdio.h>
+void main()
```

Comment:
"void main()" returns nothing. So, either change it to "int main()" or remove "return 0"